src/utils/VerificationValidationParser.cpp

```diff
@@ -153,9 +156,62 @@ void Parser::searchFinalDefense(Result* r) {
     }
 }
 
-Result* Parser::lc(const QString* terminalOutput) {
-    return nullptr; // TODO: implement
+Result* Parser::lc(const QString* cmd, const QString* terminalOutput) {
+	Result final;
+	final.terminalOutput = *terminalOutput;
+	final.cmd = *cmd;
+	Result* ptr_final = &final;
+
+	/* Convert terminal output and cmd into string for ease of parsing */
+	string s_terminalOutput = terminalOutput->toStdString();
```

Comment:
i like c++ strings too, but I think we should try to stick with QString
- should have same functionality, i think even more
- conversion to std::string takes a while and I don't think we gain any benefit from it being a c++ string (could be wrong here)

Reply:
unforunately, this means that all the functions below using c++ string functions should be changed to QString functions (can lookup "Qt5 QString", documentation is pretty good)